mdata/cache/accnt/if.go

```diff
@@ -0,0 +1,24 @@
+package cache
+
+type Accnt interface {
+	Add(string, uint32, uint64)
+	Hit(string, uint32)
+	GetEvictQ() chan *EvictTarget
+}
+
+type EvictTarget struct {
+	Metric string
+	Ts     uint32
+}
+
+func NewAccnt(maxSize uint64) Accnt {
```

Comment:
this should be called NewFlatAccnt since that's what it creates. that way it's also more explicit from the CCache constructor what kind of Accnt it creates